From 07dce79a554c7fcdd79b2d5f8907cd56489f7ec3 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Mon, 16 Oct 2017 11:56:50 +0530
Subject: [PATCH 172/211] board: freescale: ls1012a: fix RGMII tx delay issue

Recently logic to enable RGMII tx delay was changed by
below patch.
"net: phy: realtek: fix enabling of the TX-delay for RTL8211F"

Based on the patch, here we are enabling the tx delay again.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
Signed-off-by: Anjaneyulu Jagarlmudi <anji.jagarlmudi@nxp.com>
---
 board/freescale/ls1012ardb/eth.c |    2 +-
 drivers/net/pfe_eth/pfe_eth.c    |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/freescale/ls1012ardb/eth.c b/board/freescale/ls1012ardb/eth.c
index 286bc8a..c0f6306 100644
--- a/board/freescale/ls1012ardb/eth.c
+++ b/board/freescale/ls1012ardb/eth.c
@@ -62,7 +62,7 @@ int board_eth_init(bd_t *bis)
 	/* MAC2 */
 	ls1012a_set_mdio(1, miiphy_get_dev_by_name(DEFAULT_PFE_MDIO_NAME));
 	ls1012a_set_phy_address_mode(1, EMAC2_PHY_ADDR,
-				     PHY_INTERFACE_MODE_RGMII);
+				     PHY_INTERFACE_MODE_RGMII_TXID);
 
 	cpu_eth_init(bis);
 #endif
diff --git a/drivers/net/pfe_eth/pfe_eth.c b/drivers/net/pfe_eth/pfe_eth.c
index 8d8de40..4db823f 100644
--- a/drivers/net/pfe_eth/pfe_eth.c
+++ b/drivers/net/pfe_eth/pfe_eth.c
@@ -27,7 +27,7 @@ struct gemac_s gem_info[] = {
 
 		/* phy iface */
 		.phy_address = EMAC2_PHY_ADDR,
-		.phy_mode = PHY_INTERFACE_MODE_RGMII,
+		.phy_mode = PHY_INTERFACE_MODE_RGMII_TXID,
 	},
 };
 
-- 
1.7.1

